tests/functional_tests/data/hf_processors/test_gsm8k.py

```
@@ -0,0 +1,146 @@
+        example = {
+            "question": "If x = 5 & y = 3, what is x + y?",
+            "answer": "x + y = 5 + 3 = 8.\n#### 8",
+        }
+        result = process_gsm8k_example(example)
+        assert "x = 5 & y = 3" in result["input"]
+        assert result["original_answers"] == ["8"]
+
+    def test_unicode_characters(self):
+        example = {
+            "question": "Café sells 5 crépes at €2 each. What is the total in €?",
+            "answer": "5 × 2 = 10.\n#### 10",
```

Ruff (0.15.5)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)